platform/src/main/resources/application-wks.properties

```diff
@@ -0,0 +1,36 @@
+server.servlet.context-path=/
+spring.mvc.converters.preferred-json-mapper=gson
+
+logging.level.org.springframework.web.filter=DEBUG
+logging.level.org.springframework=DEBUG
+logging.level.com.moneygram.coreservices=DEBUG
+logging.level.org.stellar=DEBUG
+
+# general
+app.stellarNetworkPassPhrase=Test SDF Network ; September 2015
+app.hostUrl=http://localhost:8080
+app.languages=en
+app.horizonURI=https://horizon-testnet.stellar.org
+app.assets=assets-test.json
+app.jwtSecretKey=secret
+
+# sep-1
+sep1.enabled=true
+sep1.stellarFile=sep1/stellar-wks.toml
```

Comment:
It would be great if we could dynamically build a toml file based on the SEPs and app configurations. Maybe something for later.

Reply:
Yes, that's more to come. Sep-1 needs to be re-refactored to work like sep-10 and sep-24.